google/cloud/bigtable/row_range.h

```diff
@@ -173,6 +169,12 @@ class RowRange {
   ::google::bigtable::v2::RowRange row_range_;
 };
 
+bool operator==(RowRange const& lhs, RowRange const& rhs);
```

Comment:
Any reason why this implementation shouldn't be moved to the header file? I know it was already in the `.cc` file, but I'm wondering.

Reply:
The implementation is 37 lines which is pretty far above the 10 line threshold the style guide recommends for inlining.